lib/instrumentation.js

```diff
@@ -83,3 +73,44 @@ function wrapMiddleware(shim, middleware) {
     }
   })
 }
+
+function wrapCreateContext(shim, fn, fnName, context) {
+  // The `context.body` and `context.response.body` properties are how users set
+  // the response contents. It is roughly equivalent to `res.send()` in Express.
+  // Under the hood, these set the `_body` property on the `context.response`.
+  context.__NR_body = context.response.body
+  context.__NR_bodySet = false
+  Object.defineProperty(context.response, '_body', {
+    get: () => context.__NR_body,
+    set: function setBody(val) {
+      shim.savePossibleTransactionName(context.req)
+      context.__NR_body = val
+      context.__NR_bodySet = true
+    }
+  })
+
+  // Sometimes people just set `context.status` or `context.response.status`
```

Comment:
I don't see any code checking for `context.status`... does that set `context.response.status` under the hood?